portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -68,7 +69,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     Document doc = Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
     LanguageServiceClient languageService = LanguageServiceClient.create();
     Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
-    float score = sentiment.getScore();
+    double score = sentiment.getScore();
```

Comment:
something seems off here, in that `score` isn't being written to the datastore...

Reply:
Hmm, interesting. Seems like the remote repo was not updated. It should be fixed now.